setup.py

```diff
@@ -17,10 +17,10 @@
     description="Common functions for UK Biobank Data",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="https://github.com/Nealelab/ukb_common",
+    url="https://github.com/Nealelab/ukbb_common",
```

Comment:
@konradjk 
https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository